<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflectanceForm</class>
 <widget class="QDialog" name="ReflectanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reflectance Calcuator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Radiance Raster</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>IMU Time Offset (hrs to UTC)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Reflectance Output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spnIMUUTCOffset">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-999.999000000000024</double>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtFrameIndex"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Frame Index</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QToolButton" name="btnReflOutputFile">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spnIrradUTCOffset">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-999.999000000000024</double>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="txtConvIrrad"/>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="btnFrameIndexFile">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IMU GPS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtIMUGPS"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Convolved Irradiance</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="btnRadRastFile">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="btnConvIrradFile">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Irrad. Time Offset (hrs to UTC)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="txtReflOutput"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtRadRast"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="btnFrameIndexDir">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QToolButton" name="btnIMUGPS">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="btnRadRastDir">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="btnConvIrradDir">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QToolButton" name="btnReflOutputDir">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
